install/kubernetes/helm/istio/charts/security/templates/deployment.yaml

```diff
@@ -63,11 +63,17 @@ spec:
           {{- if .Values.workloadCertTtl }}
             - --workload-cert-ttl={{ .Values.workloadCertTtl }}
           {{- end }}
+            - --manage-unlabeled={{ .Values.manageUnlabeled }}
```

Comment:
If value is set, so it reduces the upgrade risks ( version in the repo not having the flag )

Reply:
Right, agreed. I was having a hard time figuring out how to accomplish this, since

```
          {{- if .Values.manageUnlabeled }}
            - --manage-unlabeled={{ .Values.manageUnlabeled }}
          {{- end }}
```

would actually leave the CLI flag out altogether in the case that the user set it to `false` (since Helm does not distinguish between the value not existing and the value existing as `false` in this context).

Any ideas?